docs/articles/deployment/scenarios.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: deployment-scenarios
+title: Deployment Scenarios
+---
+
+# Console Application
+
+The console application gives immediate feedback on how a piece of software works. That how most developers test out, quickly, new products to understand its entry and exit point!
+
+[Console Application](../deployment/console.html)
+
+# ASP.NET Core
+
+By using Akka.NET in your ASP.NET Core project, you can make your `Controllers` lightweight while Akka.NET handles the heavyweights.
+
+[ASP.NET Core](../deployment/aspnet-core.html)
```

Comment:
Need to follow our DocFx guidelines on using links - always use XREF

Reply:
Noted and done!